pkg/installer/Dockerfile

```diff
@@ -0,0 +1,78 @@
+# Copyright (c) 2024 Zededa, Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+# Dockerfile to build installer img initrd
+FROM lfedge/eve-alpine:c114cf1d3ea51534f061f9aa949beb6ac5c12fb3 AS build
+SHELL ["/bin/ash", "-eo", "pipefail", "-c"]
+ENV BUILD_PKGS mkinitfs grep patch make coreutils musl-dev gcc g++ perl \
```

Comment:
While building pkg/installer: `make ZARCH=arm64 pkg/installer` I've got the following errors:

```
#29 0.348 Generating file lists: src_vars.mk
#29 9.739 sh: git: not found
#29 9.747 sh: git: not found
#29 9.750 sh: git: not found
#29 9.795 sh: git: not found
#29 9.807 sh: git: not found
#29 9.811 sh: git: not found
#29 9.816 sh: git: not found
#29 9.877 sh: git: not found
#29 10.75 sh: git: not found
#29 11.36 sh: git: not found
#29 11.37 sh: git: not found
#29 19.76 aclocal: installing 'm4/ax_ac_append_to_file.m4' from '/usr/share/aclocal/ax_ac_append_to_file.m4'
#29 19.77 aclocal: installing 'm4/ax_ac_print_to_file.m4' from '/usr/share/aclocal/ax_ac_print_to_file.m4'
#29 19.79 aclocal: installing 'm4/ax_add_am_macro_static.m4' from '/usr/share/aclocal/ax_add_am_macro_static.m4'
#29 19.80 aclocal: installing 'm4/ax_add_fortify_source.m4' from '/usr/share/aclocal/ax_add_fortify_source.m4'
#29 19.81 aclocal: installing 'm4/ax_am_macros_static.m4' from '/usr/share/aclocal/ax_am_macros_static.m4'
#29 19.83 aclocal: installing 'm4/ax_check_compile_flag.m4' from '/usr/share/aclocal/ax_check_compile_flag.m4'
#29 19.84 aclocal: installing 'm4/ax_check_enable_debug.m4' from '/usr/share/aclocal/ax_check_enable_debug.m4'
#29 19.85 aclocal: installing 'm4/ax_check_gnu_make.m4' from '/usr/share/aclocal/ax_check_gnu_make.m4'
#29 19.86 aclocal: installing 'm4/ax_check_link_flag.m4' from '/usr/share/aclocal/ax_check_link_flag.m4'
#29 19.87 aclocal: installing 'm4/ax_code_coverage.m4' from '/usr/share/aclocal/ax_code_coverage.m4'
#29 19.88 aclocal: installing 'm4/ax_file_escapes.m4' from '/usr/share/aclocal/ax_file_escapes.m4'
#29 19.89 aclocal: installing 'm4/ax_is_release.m4' from '/usr/share/aclocal/ax_is_release.m4'
#29 19.91 aclocal: installing 'm4/libtool.m4' from '/usr/share/aclocal/libtool.m4'
#29 19.92 aclocal: installing 'm4/ltoptions.m4' from '/usr/share/aclocal/ltoptions.m4'
#29 19.93 aclocal: installing 'm4/ltsugar.m4' from '/usr/share/aclocal/ltsugar.m4'
#29 19.94 aclocal: installing 'm4/ltversion.m4' from '/usr/share/aclocal/ltversion.m4'
#29 19.95 aclocal: installing 'm4/lt~obsolete.m4' from '/usr/share/aclocal/lt~obsolete.m4'
#29 19.96 aclocal: installing 'm4/pkg.m4' from '/usr/share/aclocal/pkg.m4'
#29 28.39 sh: git: not found
#29 28.39 sh: git: not found
#29 28.40 sh: git: not found
#29 28.45 sh: git: not found
#29 28.46 sh: git: not found
#29 28.46 sh: git: not found
#29 28.46 sh: git: not found
#29 28.52 sh: git: not found
#29 29.36 sh: git: not found
#29 29.91 sh: git: not found
#29 29.93 sh: git: not found
#29 31.93 sh: git: not found
#29 31.94 sh: git: not found
#29 31.94 sh: git: not found
#29 31.99 sh: git: not found
#29 32.01 sh: git: not found
#29 32.01 sh: git: not found
#29 32.02 sh: git: not found
#29 32.07 sh: git: not found
#29 33.01 sh: git: not found
#29 33.77 sh: git: not found
#29 33.78 sh: git: not found
```

However, the package builds successfully, I think you could add git as well to PKGS to get rid of these messages....

Reply:
I couldn't reproduce it, but if you could, and got that message, I will add `git` to `BUILD_PKGS`.

Reply:
Done. Let's see that it is clear before we mark this conversation as "resolved"

Reply:
It worked here, the error is gone.